### PR TITLE
[release/1.1.0] Mark packages stable

### PR DESCRIPTION
**Pull request by ericstj**
This marks corefx packages stable.

**DO NOT MERGE** until we have a TFS build from ProjectK to produce the new SNI packages, otherwise the official build will fail during testing.

/cc @gkhanna79 @karelz @saurabh500 @weshaggard

### CODE REVIEW

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop OSX Debug Build and Test

**Comment by ericstj** [comment]
@dotnet-bot test Innerloop OSX Release Build and Test

**Comment by ericstj** [comment]
Should be good to go now.  TFS build completed.

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "Microsoft.CSharp": {
       "StableVersions": [
         "4.0.1",
-        "4.0.0"
+        "4.0.0",
```
Comment (weshaggard) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: I'm assuming you used some sort of tool to update the index and have verified that all the packages are stable now from a clean build.

**Comment by ericstj** [comment]
@gkhanna79 please let me know when you are ready to have this merged.

**Comment by gkhanna79** [comment]
@ericstj Will do.

**Pull request merged**